packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/information/index.scss

```diff
@@ -0,0 +1,5 @@
+.information {
+  .download-icon {
+    -webkit-text-stroke-width: 1px;
```

Comment:
**issue(blocking):** Please avoid such override.

Reply:
We can use` font-weight-bold `may be to get the same result ?

Reply:
Replaced `-webkit-text-stroke-width` with `font-weight-bold`. Though still the button doesn't match the other icons, the visibility is better.
cc- @antleblanc @varun257